coralibre-sdk/sdk/src/main/java/org/coralibre/android/sdk/internal/database/PersistentDatabase.java

```diff
@@ -0,0 +1,120 @@
+package org.coralibre.android.sdk.internal.database;
+
+import android.content.Context;
+
+import androidx.annotation.NonNull;
+import androidx.room.Room;
+
+import org.coralibre.android.sdk.internal.crypto.ppcp.CryptoModule;
+import org.coralibre.android.sdk.internal.crypto.ppcp.ENNumber;
+import org.coralibre.android.sdk.internal.crypto.ppcp.TemporaryExposureKey;
+import org.coralibre.android.sdk.internal.database.model.CapturedData;
+import org.coralibre.android.sdk.internal.database.model.GeneratedTEK;
+import org.coralibre.android.sdk.internal.database.model.IntervalOfCapturedData;
+import org.coralibre.android.sdk.internal.database.model.IntervalOfCapturedDataImpl;
+import org.coralibre.android.sdk.internal.database.model.entity.EntityCapturedData;
+import org.coralibre.android.sdk.internal.database.model.entity.EntityGeneratedTEK;
+import org.coralibre.android.sdk.internal.database.persistenDatabase.RoomDatabaseAbstrImpl;
+
+import java.util.HashMap;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+
+public class PersistentDatabase implements Database {
+
+    private final String dbName = "db";
+    private final RoomDatabaseAbstrImpl db;
+
+
+    public PersistentDatabase(@NonNull Context context) {
```

Comment:
This should have a JavaDoc comment explaining what this constructor defaults to (in-memory or not).

Reply:
Ok